frontend/src/js/controllers/analyticsCtrl.js

```diff
@@ -215,6 +213,7 @@
                 }
             } else {
                 vm.isTeamSelected = false;
+                alert("Unfortunately, there are no challenges to analyze.");
```

Comment:
Overall works well but please remove the changes from this file in this PR since it is not relevant to this PR. We can go ahead and merge this once you have done the changes.

Reply:
Okay, thank you!